check_api_breakage.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# Script: check_api_breakage.sh
```

Comment:
Is this script used 🤔 ? 
If so, should we move it to a folder instead? Maybe inside [CircleciScripts](https://github.com/aws-amplify/amplify-swift/tree/main/CircleciScripts) (outdated name is outdated)

Reply:
This script is for users that wanted to check API breakage locally. It's a local version of the yml file.